src/types.ts

```diff
@@ -17,3 +17,7 @@ export type FinancingProductType = {
 };
 
 export type FinancingPromotionLayoutType = 'full' | 'banner';
+
+export type IntervalType = 'day' | 'week' | 'month' | 'quarter' | 'year';
```

Comment:
We don't have to define these here - these can be imported from the connect-js repo